### PR TITLE
Add more information about the standalone version

### PR DESCRIPTION
**Pull request by dansku**
Change-type: minor
Signed-off-by: Daniel Andrade <daniel@balena.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ we do now have an experimental standalone binary release available, see ['Standa
 	- if you need `ssh` to work behind the proxy you also need [`proxytunnel`](http://proxytunnel.sourceforge.net/) installed (available as `proxytunnel` package for Ubuntu, for example)
   - `rsync`: >= 2.6.9 (required by `balena sync`)
 
-##### Windows Support
+#### Windows Support
+
+> Note that the fastest way to have the CLI running on **Windows** is by using the [Standalone Version](https://github.com/balena-io/balena-cli#standalone-install). You can follow [this video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA) to learn how to install it.
```
Comment (pdcastro) [inline README.md]: Very nice, @dansku ! This is good and correct but also contradicts the following paragraphs that state that the standalone install is experimental and imply that the npm install is preferred. I have thus added a commit to your branch with a rewriting of those paragraphs. Please review too! :-)

File: README.md [hunk 2]
```diff
@@ -7,42 +7,41 @@ Balena CLI
...
+Getting Started
+---------------
 
-Before installing balena-cli, you'll need a working node-gyp environment. If you don't already have one you'll see native module build errors during installation. To fix this, run `npm install -g --production windows-build-tools` in an administrator console (available as 'Command Prompt (Admin)' when pressing windows+x in Windows 7+).
+The easiest and recommended way of installing the CLI on all platforms (Linux, MacOS, Windows)
+is to use the Standalone Install described below. Some specific CLI commands like `balena ssh`
+and `balena sync` have additional dependencies: see section Additional Dependencies.
 
-`balena sync` and `balena ssh` have not been thoroughly tested on the standard Windows cmd.exe shell. We recommend using bash (or a similar) shell, like Bash for Windows 10 or [Git for Windows](https://git-for-windows.github.io/).
+> **Windows users:** we now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
```
Comment (pdcastro) [inline README.md]: @dansku, in this commit, subject to review and comments, I have moved the video link here.

Comment (dansku) [reply README.md]: Looks good to me!

File: lib/actions/push.ts [hunk 3]
```diff
@@ -44,10 +44,9 @@ function getBuildTarget(appOrDevice: string): BuildTarget | null {
 }
 
 async function getAppOwner(sdk: BalenaSDK, appName: string) {
-	const {
-		exitWithExpectedError,
-		selectFromList,
-	} = await import('../utils/patterns');
+	const { exitWithExpectedError, selectFromList } = await import(
+		'../utils/patterns'
+	);
```
Comment (pdcastro) [inline lib/actions/push.ts]: This is [prettier](https://www.npmjs.com/package/prettier)'s doing. Maybe a new version of prettier got released, or something. It was actually required in order for the Travis build to pass (as the build checks whether all source files "conform" with what prettier would do if it was run).

File: README.md [hunk 4]
```diff
@@ -7,42 +7,41 @@ Balena CLI
...
-2. Install the `msys-rsync` and `msys-openssh` packages.
-3. Add MinGW to the `%PATH%` if this hasn't been done by the installer already. The location where the binaries are places is usually `C:\MinGW\msys\1.0\bin`, but it can vary if you selected a different location in the installer.
-4. Copy your SSH keys to `%homedrive%%homepath\.ssh`.
-5. If you need `ssh` to work behind the proxy you also need to install [proxytunnel](http://proxytunnel.sourceforge.net/)
+* Download the latest zip for your OS from https://github.com/balena-io/balena-cli/releases.
+* Extract the contents, putting the `balena-cli` folder somewhere appropriate for your system (e.g. `C:/balena-cli`, `/usr/local/lib/balena-cli`, etc).
+* Add the `balena-cli` folder to your `PATH` ([Windows instructions](https://www.computerhope.com/issues/ch000549.htm), [Linux instructions](https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix), [OSX instructions](https://stackoverflow.com/questions/22465332/setting-path-environment-variable-in-osx-permanently))
+* Running `balena` in a fresh command line should print the balena CLI help. See Windows Support
```
Comment (shaunmulligan) [inline README.md]: Add link down to `See Windows Support..`

File: package.json [hunk 5]
```diff
@@ -41,7 +41,7 @@
     "pretest": "npm run build",
     "test": "gulp test",
     "test:fast": "npm run build:fast && gulp test",
-    "ci": "npm run test && (catch-uncommitted || echo 'new Date().toISOString().startsWith(\"2019-04-13\") || process.exit(1)' | node)",
```
Comment (pdcastro) [inline package.json]: This reverts what I had to do in the previous (unrelated) CLI PR in order to get the Travis build to pass. Possibly what was going wrong was that the version of [prettier](https://www.npmjs.com/package/prettier) was different between my laptop and the Travis build (we don't currently commit `package-lock.json` for the CLI).

**Pull request merged**